.github/src/shared/base.pkl

```diff
@@ -0,0 +1,52 @@
+import "@gha/Workflow.pkl"
+
+import "@gha/actions/Common.pkl"
+import "@gha/actions/Setup.pkl"
+import "@gha/actions/Artifact.pkl"
+
+baseJob: Workflow.Job = new {
+  permissions {
+    checks = "write"
+  }
+  name = "Build and test"
```

Comment:
Do we need this here? 🤔 As you overwrite it in every sub job anyways..?!

Reply:
Good point!